extensions/stackdriver/log/logger_test.cc

```diff
@@ -98,6 +99,7 @@ google::logging::v2::WriteLogEntriesRequest expectedRequest(
   (*top_label_map)["destination_name"] = node_info.name();
   (*top_label_map)["destination_workload"] = node_info.workload_name();
   (*top_label_map)["destination_namespace"] = node_info.namespace_();
+  (*top_label_map)["mesh_uid"] = node_info.mesh_id();
```

Comment:
This MESH_ID change should be a separate (small) PR.

Reply:
Done. https://github.com/istio/proxy/pull/2494